haystack/preview/components/classifiers/document_language_classifier.py

```diff
@@ -42,17 +45,15 @@ def __init__(self, languages: Optional[List[str]] = None):
         if not languages:
             languages = ["en"]
         self.languages = languages
```

Comment:
From a classifier, I would expect it to try to detect every possible language, unless the user restricts the choice (using the  `languages` variable).

Does it make sense to you?
How does it play with the `MetaDataRouter` component?

Reply:
It makes the detection much less robust if we do not limit the set of possible languages. I would expect more misclassifications without a limitation. That's the reason why I would limit it.

Reply:
For a usage example in a pipeline, see the code above in the PR description or in the docstring. 🙂